gitea/release.go

```diff
@@ -26,8 +26,9 @@ type Release struct {
 	// swagger:strfmt date-time
 	CreatedAt time.Time `json:"created_at"`
 	// swagger:strfmt date-time
-	PublishedAt time.Time `json:"published_at"`
-	Publisher   *User     `json:"author"`
+	PublishedAt time.Time     `json:"published_at"`
+	Publisher   *User         `json:"author"`
+	Attachments []*Attachment `json:"attachments"`
```

Comment:
Should I use here attachments or assets (in JSON)?

Reply:
ref https://developer.github.com/v3/repos/releases/#list-releases-for-a-repository, it should be assets